src/components/GordonConfetti/index.jsx

```diff
@@ -5,13 +5,13 @@ const GordonConfetti = ({ active, colorOption, colors }) => {
   let config = {
     angle: 90,
     spread: 360,
-    startVelocity: 40,
+    startVelocity: 20,
     elementCount: 200,
-    dragFriction: 0.12,
-    duration: 6000,
-    stagger: 3,
-    width: '10px',
-    height: '10px',
+    dragFriction: 0.001, //some numbers were adjusted for the confetti effects
+    duration: 10000,
+    stagger: 5,
+    width: '30px',
+    height: '60px',
```

Comment:
Were these changes specifically requested or approved by Chris Carlson? He originally approved the confetti mechanics as they are now. I wouldn't want to change it globally without consulting him.

Reply:
I just changed them myself when I was messing around with the confetti to see what would happen-I can change them back to their originals